helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -100,19 +73,26 @@ controller:
       cpu: 500m
       memory: 600Mi
 
+  # optional hpa settings
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 20
+    targetCPUUtilizationPercentage: 50
+    targetMemoryUtilizationPercentage: 50
+
+  # command args options
+  annotationsPrefix: nginx.ingress.kubernetes.io
+  defaultSSLCertificate: ""
+  ingressClass: nginx
+
+  # www-data -> uid 101
   userID: 101
   groupID: 101
 
 image:
   registry: quay.io
 
-global:
```

Comment:
With exception of useProxyProtocol, these weren't used anywhere, dropped them. Dropped also useProxyProtocol since it can be overriden with use-proxy-protocol configmap, and upstream defaults to false so no need to explicitly set it in configmap either.